bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -231,4 +231,13 @@
 		<category>Time</category>
 		<state readOnly="true"/>
 	</channel-type>
+
+	<channel-type id="rollershutter">
```

Comment:
Can you add information about this Channel to the README.md?

Reply:
Done.